src/model/db/AbstractModel.ts

```diff
@@ -10,8 +10,8 @@ export abstract class AbstractModel {
     };
 
     @PrimaryGeneratedColumn("increment")
-    @Name("id")
-    @Description("The id of this entry")
+    @Name("ID")
```

Comment:
`@Name` can't be changed unless you update the JS along woth it, `@Name` are bound body paerams and are are camal case.